cmd/brew-services.rb

```diff
@@ -236,6 +237,33 @@ def restart(target)
       end
     end
 
+    # "load" a plist
+    def launchctl_load(plist, _function, service)
```

Comment:
That was the intention, but I was concerned that changing the message from "started" run "ran" in the `brew run` case might break callers. So I wanted to make the API for the function call support that concept without commiting to it. Which way should I go? Is it OK to change the output for the `brew run` case?

(On the other hand, maybe `brew run` really does `start` things instead of `running` them. In which case this should all just go away?)